Automattic/wp-calypso
# SupportArticleDialog: migrate CSS to webpack

Straightforward migration. I kept the separate `content.scss` stylesheet, only removed the `_` prefix that's not needed when SASS files are processed by webpack.

**How to test:**
Open the Inline Help popup and display any support article. Verify the styles of the modal that opens.
